sdk/include/psdk/windot11.h

```diff
@@ -34,6 +34,17 @@ typedef enum _DOT11_AUTH_ALGORITHM {
     DOT11_AUTH_ALGO_WPA_NONE,
     DOT11_AUTH_ALGO_RSNA,
     DOT11_AUTH_ALGO_RSNA_PSK,
+    DOT11_AUTH_ALGO_WPA3,
+#if (NTDDI_VERSION >= NTDDI_WIN10_FE)
+    DOT11_AUTH_ALGO_WPA3_ENT_192       = DOT11_AUTH_ALGO_WPA3,
+#endif
+    DOT11_AUTH_ALGO_WPA3_SAE,
+#if (NTDDI_VERSION >= NTDDI_WIN10_VB)
+    DOT11_AUTH_ALGO_OWE,
+#endif
+#if (NTDDI_VERSION >= NTDDI_WIN10_FE)
+    DOT11_AUTH_ALGO_WPA3_ENT,
```

Comment:
Because of the jumps introduced with the versioning in the enum values, you need to explicitly specify the values of the following ones:
```suggestion
    DOT11_AUTH_ALGO_WPA3_SAE           = 9,
#if (NTDDI_VERSION >= NTDDI_WIN10_VB)
    DOT11_AUTH_ALGO_OWE                = 10,
#endif
#if (NTDDI_VERSION >= NTDDI_WIN10_FE)
    DOT11_AUTH_ALGO_WPA3_ENT           = 11,
```

Reply:
> Because of the jumps introduced with the versioning in the enum values

There can't be any jumps. If `NTDDI_VERSION` is less than `NTDDI_WIN10_FE`, the deprecated enumeration name (which we chose to **keep** as it's a part of public SDK) of `DOT11_AUTH_ALGO_WPA3` retains `8` value and `DOT11_AUTH_ALGO_WPA3_SAE` will be `9`, as expected.